test/functorch/test_aotdispatch.py

```diff
@@ -960,7 +960,8 @@ def f(a, b, c):
         inp = [torch.randn(5, requires_grad=True) for _ in range(3)]
         f(*inp).sum().backward()
 
-    def test_compilation_context(self):
+    @patch('functorch._src.aot_autograd.AOT_COUNTER', new_callable=itertools.count)
```

Comment:
maybe resetting the context should just be in a unittest `teardown()`?

Reply:
patch automatically undos the patch on exit